tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -16,9 +17,40 @@ class TestVmSnapshotClone(E2ETest):
     """
 
     @workloads
-    @pytest.mark.polarion_id("OCS-6288")
+    @pytest.mark.parametrize(
+        argnames=["pvc_expand_before_clone", "pvc_expand_after_clone"],
+        argvalues=[
+            pytest.param(
+                False,
+                False,
+                marks=pytest.mark.polarion_id(
+                    "OCS-6288"
+                ),  # Polarion ID for no PVC expansion
+            ),
+            # pytest.param(
```

Comment:
Please check if you can skip expansion test for now with the CNV bug you filed.

Reply:
I have refactored code of multi_cnv_workload fixture where I am creating non encryption pvc till this bug get resolved.